pandas/plotting/api.py

```diff
@@ -0,0 +1,20 @@
+"""
+Plotting api
```

Comment:
I know this is a bit the style in other pandas modules as well, but can't we just put this in `__init__.py` ?